cddl/trust_domains.cddl

```diff
@@ -4,7 +4,7 @@
 ;
 ; Removed unused extensions
 ; Set the maximum number of parameters inside suit-directive-set-parameters to 6
-; Limited the number of possible dependency manifests to 4
+; Limited the number of possible dependency manifests to 6
```

Comment:
I think it should be increased to 7. That is the amount of dependency manifests  which could be declared by APP_Recovery manifest.

Reply:
Increased `SUIT_Dependencies` to 7.